src/services/YotiService.ts

```diff
@@ -137,9 +138,12 @@ export class YotiService {
 		countryCode: string,
 		yotiCallbackUrl: string,
 	): Promise<string | undefined> {
+		const sessionDeadlineDate = new Date(new Date().getTime() + (this.YOTI_SESSION_TTL_DAYS ? this.YOTI_SESSION_TTL_DAYS : 10) * 24 * 60 * 60 * 1000);
+		sessionDeadlineDate.setHours(22, 0, 0, 0);
+
 		const payloadJSON: CreateSessionPayload = {
-			client_session_token_ttl: this.CLIENT_SESSION_TOKEN_TTL_SECS ? this.CLIENT_SESSION_TOKEN_TTL_SECS : "950400",
-			resources_ttl: this.RESOURCES_TTL_SECS ? this.RESOURCES_TTL_SECS : "1036800",
+			session_deadline: sessionDeadlineDate,
+			resources_ttl: this.RESOURCES_TTL_SECS ? this.RESOURCES_TTL_SECS : "1209600",
```

Comment:
is resources_ttl: this.RESOURCES_TTL_SECS ? this.RESOURCES_TTL_SECS : "1209600" required as EnvironmentVariables,ts should be setting it to default 14 days in secs if not present?

Reply:
Removed check in YotiService